src/server/hotspot_partition_calculator.h

```diff
@@ -39,15 +46,19 @@ class hotspot_partition_calculator
     void data_analyse();
 
 private:
-    const std::string _app_name;
+    // 3σ rule to calculate hot point of each partition
```

Comment:
What is `3σ` means?

Reply:
https://en.wikipedia.org/wiki/68%E2%80%9395%E2%80%9399.7_rule

Reply:
I think `empirical rule` may be better than `3σ rule`, besides you should add explanation for this statistics rule or add link here, the rule name will just make me confused.